xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -8,132 +8,146 @@
+| ardana002-cp1-comp0002-mgmt | 55c93eb7d609467a8287f175a2275219 | ardana002-cp1-comp0002-mgmt |             |
+| ardana002-cp1-c0-m1-mgmt    | 50d26318e81a408e97d1b6639b9404b2 | ardana002-cp1-c0-m1-mgmt    |             |
+| ardana002-cp1-c1-m1-mgmt    | 78fe921473914bf6a802ad360c09d35b | ardana002-cp1-c1-m1-mgmt    |             |
+| ardana002-cp1-c1-m2-mgmt    | b2e9a4305c4b4272acf044e3f89d327f | ardana002-cp1-c1-m2-mgmt    |             |
+| ardana002-cp1-c1-m3-mgmt    | a3ceb80b8212425687dd11a92c8bc48e | ardana002-cp1-c1-m3-mgmt    |             |
++-----------------------------+----------------------------------+-----------------------------+-------------+</screen>
+    <para>
+     In this example, the client id of host
+     <literal>ardana002-cp1-c1-m1-mgmt</literal> is
+     <literal>ardana002-cp1-c1-m1-mgmt</literal>.
+    </para>
+   </step>
+   <step>
     <para>
      List the jobs
     </para>
-<screen>freezer-scheduler -c &lt;hostname&gt; job-list</screen>
-   </listitem>
-   <listitem>
+<screen>freezer job-list -C ardana002-cp1-c1-m1-mgmt</screen>
+   </step>
+   <step>
     <para>
-     Get the id corresponding to the job <literal>&lcm; Default: Mysql restore
+     Get the corresponding job id for <literal>Ardana Default: mysql restore
      from Swift</literal>.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Launch the restore.
+     Launch the restore process with <literal>freezer job-start</literal>
+     <replaceable>JOB-ID</replaceable>.
     </para>
-<screen>freezer-scheduler -c &lt;hostname&gt; job-start -j &lt;job-id&gt;</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     This will take some time. While you wait you can follow the progress in
-     the <literal>/var/log/freezer-agent/freezer-scheduler.log</literal> log
-     file.
+     This will take some time. You can follow the progress in
+     <literal>/var/log/freezer/freezer-scheduler.log</literal>.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Log in to the &lcm;.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Stop the Percona DB service:
+     Log back in to the first node running the &mariadb; service, the same node
+     as in <xref linkend="mariadb-nodes"/>.
     </para>
-<screen>cd ~/scratch/ansible/next/ardana/ansible
-ansible-playbook -i hosts/verb_hosts percona-stop.yml</screen>
-   </listitem>
-   <listitem>
-    <para>
-     Log back in to the first node running the &mariadb; service. (Same node as
-     step #3)
-    </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Clean the &mariadb; directory using this command:
     </para>
 <screen>sudo rm -r /var/lib/mysql/*</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Copy the restored files back to the &mariadb; directory:
```

Comment:
check operationswiftfix3, ask user to wait until restore job is done to do this step